ecl/ecl-bundle/CMakeLists.txt

```diff
@@ -54,9 +55,10 @@ include_directories (
          ${HPCC_SOURCE_DIR}/ecl/eclcmd
          ${HPCC_SOURCE_DIR}/ecl/eclcc
          ${HPCC_SOURCE_DIR}/common/thorhelper
+         ${ESPSCM_GENERATED_DIR}
     )
 
-ADD_DEFINITIONS( -D_CONSOLE -DWSSMC_API_LOCAL )
+ADD_DEFINITIONS( -D_CONSOLE -DWSSMC_API_LOCAL -DFileSpray_API_LOCAL )
```

Comment:
This suggests the existing structure isn't quite right.  Why would the ecl-bundle command have a dependency on the spray interface?

Reply:
Agreed.  Need to look at when/how EclCmdOptionsDFU is incorporated.  

I've opened JIRA https://track.hpccsystems.com/browse/HPCC-29460